_projects/27-trait-init-encoding.md

```diff
@@ -8,9 +8,10 @@ contributors: [sjrd]
 status: Completed
 type: project
 active: true
-priority: 1
+priority: -1
```

Comment:
this will make it below the other projects that are active in scala3migration category